src/measure_theory/measure/measure_space.lean

```diff
@@ -1077,6 +1082,101 @@ lemma comapₗ_eq_comap {β} [measurable_space α] {mβ : measurable_space β} (
   comapₗ f μ s = comap f μ s :=
 (comapₗ_apply f hfi hf μ hs).trans (comap_apply f hfi hf μ hs).symm
 
+
+section Yael's_application
```

Comment:
I don't think we want to keep this name. Feel free to document the application in a module docstring.

Reply:
Indeed that name must go. And I need to move some lemmas, etc. That code is not yet ready for review.